crypto/sleep.c

```diff
@@ -44,13 +44,34 @@ void OSSL_sleep(uint64_t millis)
 
 void OSSL_sleep(uint64_t millis)
 {
-    Sleep(millis);
+    /*
+     * Windows' Sleep() takes a DWORD argument, which is smaller than
+     * a uint64_t, so we need to split the two to shut the compiler up.
+     */
+    DWORD dword_times = (DWORD)(millis >> sizeof(DWORD));
```

Comment:
And this should be shifted  `8 * sizeof(DWORD)` shouldn't it?

Reply:
Ah, yes!  Thanks, fixed